src/sagas/config.js

```diff
@@ -8,12 +8,15 @@ import { refreshConfig } from 'actions';
 
 
 function* fetchConfig({uid, userData}) {
-    window._FIREBASE_DB_.ref('/CONFIG')
-        .on('value', (snapshot) => {
-            const config = snapshot.val();
+
```

Comment:
what config are we planning on getting exactly?

Reply:
Currently: ![image](https://user-images.githubusercontent.com/857676/45102308-66a26a00-b0fb-11e8-8af9-9f40a6203f8c.png)

For other DBs, it included nodes for items such as: GLOBAL_ALERT_MESSAGE, FEATURE1_ENABLED, etc...